scripts/migrations/2025_01_06_addMixpanelSignupEvents.ts

```diff
@@ -0,0 +1,57 @@
+import { log } from '@charmverse/core/log';
+import { prisma } from '@charmverse/core/prisma-client';
+import { batchImportMixpanelEvent } from '@packages/mixpanel/updateUserProfile';
+import { uuidFromNumber } from '@packages/utils/uuid';
+import fs from 'node:fs';
+
+async function trackMixpanelSignupEvents(userIds: string[]) {
+  const scouts = await prisma.scout.findMany({
+    where: {
+      id: {
+        in: userIds
+      },
+      onboardedAt: {
```

Comment:
i think we should stick with our logic, which means that sign_up events occur before you are onboarded and being onboarded is not required

Reply:
done